pulp_docker/app/tasks/sync_stages.py

```diff
@@ -180,22 +210,17 @@ def __init__(self, remote):
             extra_data={'headers': V2_ACCEPT_HEADERS}
         )
         list_dc = DeclarativeContent(content=manifest_list, d_artifacts=[da])
-        for manifest in manifest_list_data.get('manifests'):
-            await self.create_pending_manifest(list_dc, manifest)
-        list_dc.extra_data['relation'] = tag_dc
-        list_dc.extra_data['processed'] = True
-        tag_dc.extra_data['processed'] = True
-        await self.put(list_dc)
-
-    async def create_and_process_tagged_manifest(self, tag_dc, manifest_data):
+
+        return list_dc
+
+    def create_and_process_tagged_manifest(self, tag_dc, manifest_data):
         """
         Create a Manifest and nested ManifestBlobs from the Tag artifact.
```

Comment:
No longer creates nested ManifestBlobs

Reply:
Btw, this is what I meant by "process", so maybe the name of the function should be changed too.